### PR TITLE
MEPTS-461-Fix C1 query

### PR DESCRIPTION
**Pull request by amosh101**

### CODE REVIEW

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java [hunk 1]
```diff
@@ -44,14 +44,15 @@ public CohortDefinition getA() {
...
-    cd.addSearch("transferredIn", map(transferredIn, less12m));
+    cd.addParameter(new Parameter("onOrAfter", "onOrAfter", Date.class));
+    cd.addParameter(new Parameter("onOrBefore", "onOrBefore", Date.class));
+    cd.addParameter(new Parameter("location", "location", Location.class));
+    cd.addSearch("startedArt", mapStraightThrough(startedArt));
+    cd.addSearch(
+        "transferredIn",
+        map(
+            transferredIn,
+            "onOrAfter=${onOrAfter},onOrBefore=${onOrBefore+12m},location=${location}"));
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: So you are looking into 1 year a head of the end of reporting period?

Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: onOrBefore=${onOrBefore+12m}

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java [hunk 2]
```diff
@@ -799,32 +799,40 @@ public CohortDefinition getNumberOfPatientTbScreenedInFirstEncounter() {
...
+            + "                                 AND e.location_id = :location "
+            + "                                 AND e.voided = 0 "
+            + "                                 AND p.voided = 0 "
+            + "                                  AND o.voided = 0 "
+            + "                            AND e.encounter_datetime BETWEEN :startDate AND :endDate  "
+            + "                           GROUP BY p.patient_id) minDate "
+            + "                          INNER JOIN encounter enc ON minDate.patient_id = enc.patient_id  "
+            + "                          WHERE minDate.min_date = enc.encounter_datetime) AS encounters "
+            + "                         ON encounters.encounter_id = form.encounter_id "
+            + "                         WHERE form.patient_id is not null"
```
Comment (ningosi) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java:834]: Make this **is not null**  to **IS OT NULL** like everywhere else you have considered key and preserved words

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java (hunk 1 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: This is pertaining the changes Yassin is doing to the cohort report period, i merged develop into this PR so it was included. We will need to reachout to him for further clarifications

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 2 above)
Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Done.

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java (hunk 1 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: Ok, that is clear

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java (hunk 2 above)
Comment (ningosi) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoMensalCohortQueries.java]: Thanks

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java (hunk 1 above)
Comment (ynurmahomed) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: @amosh101 @ningosi  Guys this commit is reverting the changes done in #45. Please undo.

Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: Undone, please confirm.

Comment (amosh101) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/ResumoTrimestralCohortQueries.java:55]: @ynurmahomed Please refer to https://github.com/jembi/openmrs-module-eptsreports/pull/47/files#r413873192,

**Pull request merged**